### PR TITLE
Add relevant links to user dropdown

### PR DESCRIPTION
**Pull request by eatonphil**
Kinda makes the sidebar look ugly in the meantime. But this will help move the process along.

screen shot 2017-03-27 at 12 28 09 am

### CODE REVIEW

File: scss/components/Header.scss [hunk 1]
```diff
@@ -132,19 +132,22 @@ $header-padding: $grid-gutter-width-base / 2;
     }
 
     &-body {
+        /* Places the popup underneath the header so (among other things) the top box shadow is hidden */
+        z-index: -1;
```
Comment (mparke) [inline scss/components/Header.scss]: should be able to accomplish the same thing by just adjusting the box-shadow slightly, take a look at `0px 2px 3px 1px #DDDDDD`

**Comment by mparke** [comment]
Looks great, btw

**Comment by na3d** [comment]
I would like us to discuss breadcrumbs more, since we are going in this direction, or some solution for getting back to the top layer of those sections, without having to use the dropdown.

**Comment by eatonphil** [comment]
Already a thing:

https://github.com/linode/manager/pull/1426

**Comment by eatonphil** [comment]
But I see now that I missed the Support link in the dropdown. Will add

**Comment by eatonphil** [comment]
thank you!

**Pull request merged**